__tests__/app.test.js

```diff
@@ -40,3 +41,35 @@ describe("/api/categories", () => {
 			});
 	});
 });
+describe("/api/reviews", () => {
+	test("GET 200 - should return with all the reviews", () => {
+		return request(app)
+			.get("/api/reviews/")
+			.expect(200)
+			.then((result) => {
+				expect(result.body.length > 0);
```

Comment:
This will still need to be asserted with a .toBe(true) to confirm that this statement is actually true.